lib/src/data/datasource/local/DAOs/firestore_database.dart

```diff
@@ -19,6 +19,53 @@ class FirestoreDatabase {
     batch.commit();
   }
 
+  Future<List<Map<String, dynamic>>> getFavorites() async {
+    final List<Map<String, dynamic>> favorites = [];
+    try {
+      final QuerySnapshot snapshot = await _firestore
+          .collection('cards')
+          .where('favorite', isEqualTo: true)
```

Comment:
Trailling comma and add collection to constants

Reply:
Done